core/controllers/topics_and_skills_dashboard.py

```diff
@@ -172,8 +177,36 @@ def post(self):
 
         skill.update_explanation(
             state_domain.SubtitledHtml.from_dict(explanation_dict))
+
+        image_filenames = skill_services.get_image_filenames_from_skill(skill)
+
         skill_services.save_new_skill(self.user_id, skill)
 
+        image_validation_error_message_suffix = (
+            'Please go to oppia.org/skill_editor/%s to edit '
+            'the image.' % skill.id)
+        for filename in image_filenames:
```

Comment:
From line 188 to 208: Can we move this to a re-usable method so that we can use it in questions and suggestion as well? [Not sure!]

Reply:
All the functions that actually do the validations and save images are already in functions, these lines just call those functions and throw errors, which I think is fine to be in a controller, right?

Reply:
I'll do it as per my requirement in my PR. Thanks for the info! :)